setup-scripts/support/authorized_keys

```diff
@@ -7,3 +7,4 @@ ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIG67M3zC4eDJEjma0iKKksGclteKbB86ONQtBaWY93M6
 ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIF+XC8f0dumhzDE93i9IIMsMp7/MJPwGH+Uc9JFKOvyw karol@siasky.net
 ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIPM43lzbKjFLChe5rKETxDpWpNlqXCGTBPiWlDN2vlLD pj@siasky.net
 ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIN6Kcx8yetova4/ALUQHigo/PBMJO33ZTKOsg2jxSO2a user@deploy.siasky.dev
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQD4kuy21/2hP34dRE8uCsL23cuhtWSacOSr7NgGiEWXlydf0pzi/sjn3vBWj/WpJYhocS5nhalyy7S2gXt1ghdQngb9WoEMRJh/thioEntLKx/KlTpLqp+PLeMqXwN2twLGkcJsSQWLFYnomdsg7d3hydrtOXjGilar/g1AZjXYDVikdQAr9B98pZ4WNvUVZK1hKLlGtd0ZTS1Ab1fcPN5oe4QTNv+4mmobQQgW0jt/g9sJrQM03bwQr9hERYSTVWgSkBrykeXwu6fSbUqwPixvHWxyRIhL4ymT/B57Av9mrQxr4mWyZIN+qjNAOUi3/zFOx8iG5757i9DlhFoGv7VRz2EpolXAj6g6V6/zCFysKYBgLKJtaia7HvLyxwrivOwi5VVfpna47ZyIL7ULpLqpKAdV1v+XlMmhKg93RFAOjehBre33KJj1+qw2Qn9c8qaXADZnklk7fO/MGf19xYfAQ5XSUuPHmHJ5bOvattUFzWMd/8RNJLhjXlVa/0+a8K8IS6pWxqaLm3gAa0D3hsqorKjqSi7qB8p5DFxTogB4or12M49tiH3DPP4x5kQvehr07HaLiNo80vyPRGMDmeKcgnaiX5Atr5LmSbAG9TWk/cS7sK1u6pRYI0AJHTlXIfqUYjX6FZCqhhGeQ81DOsKvhpE+fd99/7NLobLTaY0rsw== marcin@Marcins-MacBook-Pro.local
```

Comment:
can you generate ssh-ed25519 instead of ssh-rsa ? https://medium.com/risan/upgrade-your-ssh-key-to-ed25519-c6e8d60d3c54

Reply:
Done.